lua/wikis/tarkovarena/Infobox/Person/Player/Custom.lua

```diff
@@ -0,0 +1,30 @@
+---
+-- @Liquipedia
+-- wiki=tarkovarena
+-- page=Module:Infobox/Person/Player/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local TeamHistoryAuto = require('Module:TeamHistoryAuto')
+
+local Injector = Lua.import('Module:Widget/Injector')
+local Player = Lua.import('Module:Infobox/Person')
+
+---@class TarkovArenaInfoboxPlayer: Person
+local CustomPlayer = Class.new(Player)
+local CustomInjector = Class.new(Injector)
+
+---@param frame Frame
+---@return Html
+function CustomPlayer.run(frame)
+	local player = CustomPlayer(frame)
+
+	player.args.history = TeamHistoryAuto.results{convertrole = true}
```

Comment:
You likely also want this, makes it pull the `|team` parameter from transfers as well
```suggestion
	player.args.history = TeamHistoryAuto.results{convertrole = true}
	player.args.autoTeam = true
```